services/payment-service/routes/payment/index.route.ts

```diff
@@ -4,12 +4,21 @@ import { PaymentActions } from "../../actions/payment/payment.action";
 const router = express.Router();
 const paymentActions = new PaymentActions();
 
+router.get("/", async (req, res) => {
```

Comment:
Please check my comment in [payment.action.ts](https://github.com/Digital-Envision/express-services/pull/2/commits/285c196575aa67f3565a8a325544ff3838c6b662#diff-b8595e2055e000ea41c7c72c1dbdb1baf5551638bce3937736993799d3bbc012) regarding the code standard.